google/cloud/security/scanner/scanners/firewall_rules_scanner.py

```diff
@@ -25,13 +25,13 @@
 from google.cloud.security.common.data_access import firewall_rule_dao
 from google.cloud.security.common.gcp_type import resource as resource_type
 from google.cloud.security.common.gcp_type import resource_util
-from google.cloud.security.scanner.audit import fw_rules_engine
+from google.cloud.security.scanner.audit import firewall_rules_engine
 from google.cloud.security.scanner.scanners import base_scanner
 
 LOGGER = log_util.get_logger(__name__)
 
 
-class FwPolicyScanner(base_scanner.BaseScanner):
+class FirewallPolicyScanner(base_scanner.BaseScanner):
     """Scanner for firewall data."""
 
     SCANNER_OUTPUT_CSV_FMT = 'scanner_output_fw.{}.csv'
```

Comment:
What about this?  If we are doing this renaming, we should get everything.

Reply:
Good point, I'll fix that.